<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsRangeConfigDlgBase</class>
 <widget class="QWidget" name="QgsRangeConfigDlgBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="rangeLabel">
     <property name="text">
      <string>Allows setting of numeric values from a specified range. The edit widget can be either a slider or a spin box.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QComboBox" name="rangeWidget"/>
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer_13">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <layout class="QGridLayout" name="gridRangeLayout">
     <item row="0" column="1" rowspan="3">
      <widget class="QStackedWidget" name="rangeStackedWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="intPage">
        <layout class="QVBoxLayout" name="verticalLayout_11">
         <item>
          <widget class="QSpinBox" name="minimumSpinBox">
           <property name="minimum">
            <number>-999999999</number>
           </property>
           <property name="maximum">
            <number>999999999</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="maximumSpinBox">
           <property name="minimum">
            <number>-999999999</number>
           </property>
           <property name="maximum">
            <number>999999999</number>
           </property>
           <property name="value">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="stepSpinBox">
           <property name="maximum">
            <number>999999999</number>
           </property>
           <property name="value">
            <number>1</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="doublePage">
        <layout class="QVBoxLayout" name="verticalLayout_12">
         <item>
          <widget class="QDoubleSpinBox" name="minimumDoubleSpinBox">
           <property name="minimum">
            <double>-999999999.990000009536743</double>
           </property>
           <property name="maximum">
            <double>999999999.990000009536743</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="maximumDoubleSpinBox">
           <property name="minimum">
            <double>-999999999.990000009536743</double>
           </property>
           <property name="maximum">
            <double>999999999.990000009536743</double>
           </property>
           <property name="value">
            <double>5.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="stepDoubleSpinBox">
           <property name="maximum">
            <double>999999999.990000009536743</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="stepLabel">
       <property name="text">
        <string>Step</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="maximumLabel">
       <property name="text">
        <string>Maximum</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="minimumLabel">
       <property name="text">
        <string>Minimum</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="valuesLabel">
     <property name="text">
      <string>Local minimum/maximum = 0/0</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
